pom.xml

```diff
@@ -237,7 +237,7 @@
                 </excludes>
                 <inlineHeader><![CDATA[
 Copyright Terracotta, Inc.
-Copyright Super iPaaS Integration LLC, an IBM Company 2024
+Copyright IBM Corp. 2024, 2025
```

Comment:
This commits us to a re-release of this every year right? Not saying we fix that now... but it might be something to look at in the future.